src/hipscat/catalog/association_catalog/partition_join_info.py

```diff
@@ -136,14 +137,15 @@ def read_from_dir(cls, catalog_base_dir: FilePointer, storage_options: dict = No
         """
```

Comment:
We need to update the docstring to match the change on file preference. Something like this?

> This will look for a `partition_join_info.csv` file, and if not found, will look for a  `_metadata` file. The second approach is typically slower therefore a warning is issued to the user.

Reply:
Good catch! Remind me to talk to you about PR "suggestion" mode.